pkgs/development/compilers/idris2/default.nix

```diff
@@ -0,0 +1,54 @@
+# racket is busted: https://github.com/NixOS/nixpkgs/issues/11698
+
+{ stdenv, fetchFromGitHub, makeWrapper, runCommand
+, idris, chez, racket, chickenPkgs
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Is there a specific reason you did not use the `build-idris-package` function here?

Reply:
I wasn't sure how to make this work with build-idris-package.  E.g. idris2 tests are in their own ipkg called from the makefile; buildInputs might be set incorrectly (idris should only be a nativeBuildInput for idris2 I think?); etc.